Packs/DemistoRESTAPI/Scripts/GetIncidentTasks/GetIncidentTasks.py

```diff
@@ -0,0 +1,147 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+from typing import Any
+
+
+TASK_STATES = {
+    'new': '',
+    'inprogress': 'inprogress',
+    'completed': 'Completed',
+    'waiting': 'Waiting',
+    'error': 'Error',
+    'looperror': 'LoopError',
+    'skipped': 'WillNotBeExecuted',
```

Comment:
After I spoke with @AdiPeret, we decided that we should add the 'WillNotBeExecuted' as a possible value.

```suggestion
    'skipped': 'WillNotBeExecuted',
    'WillNotBeExecuted': 'WillNotBeExecuted',
```

Reply:
OK, i'm adding it